Cython/Compiler/Nodes.py

```diff
@@ -8370,6 +8378,339 @@ def annotate(self, code):
         self.body.annotate(code)
 
 
+class StarExceptHelperNode(StatListNode):
+    """
+    Uses a stat list node for implementation (since it's
+    mostly implemented in terms of a generated tree of other nodes).
+    However, also handles some of
+    the temporary variables that its children will need.
+
+    The advantage of this is that approach is that a lot can be handled
+    by the existing flow control structures
+    """
```

Comment:
We have lots of "helper" nodes that are not called "helpers" because they do more than just helping. This one basically implements an `except*` construct, so let's call it that.
```suggestion
class ExceptStarNode(StatListNode):
    """
    Implements an 'except*' handler (Python 3.11+ / PEP-654).
    See https://docs.python.org/3/reference/compound_stmts.html#except-star

    Uses a stat list node for implementation (since it's
    mostly implemented in terms of a generated tree of other nodes).
    However, also handles some of
    the temporary variables that its children will need.

    The advantage of this is that approach is that a lot can be handled
    by the existing flow control structures
    """
```